opencog/atomspace/AtomSpace.cc

```diff
@@ -99,6 +99,156 @@ AtomSpace& AtomSpace::operator=(const AtomSpace&)
          "AtomSpace - Cannot copy an object of this class");
 }
 
+bool AtomSpace::compare_atomspaces(const AtomSpace& space_first,
+                                   const AtomSpace& space_second,
+                                   bool check_truth_values,
+                                   bool emit_diagnostics)
+{
```

Comment:
Shouldn't this be a part of some unit test, instead of being here, in the main namespace?